lib/faker/creature/bird.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+module Faker
+  class Creature
+    class Bird < Base
+      flexible :bird
+
+      class << self
+        ##
+        # Produces a random common family name of a bird
```

Comment:
Can you add a period at the end of the sentence?

```suggestion
        # Produces a random common family name of a bird.
```

Reply:
Can you update the similar parts of the other generators in this PR?